src/Uno.UI/UI/Xaml/FontFamily,wasm.cs

```diff
@@ -0,0 +1,58 @@
+﻿#if __WASM__
```

Comment:
You don't need this `#if` when the file is using the `.wasm.cs` prefix...

Ooohhh...  your file is using a comma instead of a dot!!

Reply:
🤣🤣🤣🤣

Reply:
We in Czech Republic like our commas 😂